docker-compose.yml

```diff
@@ -4,10 +4,15 @@ services:
     node:
         build:
             context: ./
+            dockerfile: ./Dockerfile
         image: wmde/wikibase-vuejs-components 
         volumes:
             - '~/.npm:/.npm'
+            - './../..:/src:ro' # grant read access to allow e.g. jest watch to detect changes
```

Comment:
This going two directories up doesn't make that much sense here, as we are already in the root directory

Reply:
Done